Sources/PackageModel/UserToolchain.swift

```diff
@@ -455,6 +457,10 @@ public final class UserToolchain: Toolchain {
     self.extraFlags.swiftCompilerFlags = try Self.deriveSwiftCFlags(
       triple: triple, destination: destination, environment: environment)
 
+    self.librarianPath = try UserToolchain.determineLibrarian(
```

Comment:
we normally format each argument in its own line

Reply:
This also seems to be using 2 spaces, while the rest of the codebase uses 4

Reply:
This was what `swift format` did 🤷‍♂️

Reply:
Reformatted with SwiftFormat rather than swift-format.